app/src/main/java/com/martynaskairys/wallpee/testing/Alarm.java

```diff
@@ -0,0 +1,78 @@
+package com.martynaskairys.wallpee.testing;
+
+import android.app.AlarmManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.PowerManager;
+import android.os.Vibrator;
+import android.telephony.TelephonyManager;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.martynaskairys.wallpee.ExitAppActivity;
+import com.martynaskairys.wallpee.MainActivity;
+import com.martynaskairys.wallpee.R;
+import com.martynaskairys.wallpee.RandomlyChangeWallpapers;
+import com.martynaskairys.wallpee.WallpaperService;
+
+public class Alarm extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+
+
+        // For our recurring task, we'll just display a message
+        Toast.makeText(context, "I'm running and... what am I suppose to do now? ", Toast.LENGTH_LONG).show();
+
+        //and let phone vibrate
+        Vibrator vibrator = (Vibrator) context.getSystemService(Context.VIBRATOR_SERVICE);
+        vibrator.vibrate(2000);
+
+
+        Intent serviceIntent = new Intent (context, WallpaperService.class);
+        context.startService(serviceIntent);
+
+
+
+       // RandomlyChangeWallpapers rcw = new RandomlyChangeWallpapers();
```

Comment:
commented out code pollutes code base, in my humble opinion :) I would remove all of it